src/app/_workflows/submitter-workflow.service.ts

```diff
@@ -0,0 +1,76 @@
+import { Injectable } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+import { DateService } from '../_services/date-service/date-service';
+import { RegionModel } from '../_models/region.model'
+import { ViewSittingRecordPost } from '../_models/viewSittingRecords.model';
+import { ViewSittingRecordService } from '../_services/sitting-records-service/view-sitting-records-service';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SubmitterWorkflowService {
+
+  formData!: FormGroup;
+  userFormData!: FormGroup;
+  hasVisitedManage = false;
```

Comment:
Consider changing this to something more like 'landingVisited' so its differentiated from other workflows

Reply:
Done